x/emissions/keeper/keeper.go

```diff
@@ -1289,34 +1272,21 @@ func (k *Keeper) RemoveDelegateStake(
 	}
 	stakeUponReputerNew := stakeUponReputer.Sub(stake)
 
-	// types updates -- done all at once after all checks
-
 	// Set new stake from delegator
 	if err := k.SetStakeFromDelegator(ctx, topicId, delegator, stakeFromDelegatorNew); err != nil {
+		fmt.Println("Setting stake from delegator failed")
 		return err
```

Comment:
We will need to revisit these Prints at some point, not sure how this is useful apart from internal tests.

Reply:
prob a better strategy is wrapping the errors instead `errors.Wrap()` from `cosmossdk.io/errors` lib

Reply:
if the chain suddenly halts, I think we're gonna be super happy we have them